README.md

```diff
@@ -835,10 +835,36 @@ function quux () {}
 
 Reports invalid padding inside JSDoc block.
 
+<a name="eslint-plugin-jsdoc-rules-check-indentation-options-1"></a>
+#### Options
+
+This rule has an object option.
+
+<a name="eslint-plugin-jsdoc-rules-check-indentation-options-1-excludeexamples"></a>
+##### <code>excludeExamples</code>
```

Comment:
I think this fix does not need to be under an option. As there is no case where `check-indent` in `@example` block is welcomed.

Reply:
There could be some rare exceptions when there is explanatory text or if one wanted the formally specified `@example` `<caption>` inside tag to be indented, but I'd tend to agree it'd be unusual and even then, it wouldn't work with the sample code following (I guess one could check the `<caption>` is indented though, as we know that to be regular text).

Also, FWIW, I've reopened #211 as the intent there is not merely to disable the ESLint `indent` rule for JavaScript within `@example` but to parse indenting correctly in our rule so that `indent` can be applied.

Reply:
I still do not find a use case for the option `excludeTags`. Adding an option is an expensive thing so unless there is a solid use case it shouldn't be added.

The `@example <caption>...` use case confuses me. Isn't it that you need `ignoreExample: true` to properly indent that case?

Reply:
@golopot There's also `@description` that allows "freestyle" text. There are also `@classdesc` and `@summary`, but i don't know if they allow multiple lines of text. Who know what else will be added in future?
I can see now that one could want to enforce simplest, almost one-liner-like examples in JSDoc, with additional `@see` links to full-blown example code in separate, "executable" files. In which case running `check-indentation` on them could help.

As for `@example <caption>...</caption>` case: i don't know, i never used captions before. I find its syntax... weird/out-of-place compared to the rest of the JSDoc syntax ;).

Reply:
Okay, that's fair.

Reply:
Thanks :).

Reply:
Sorry for the delayed reply... The idea with `@example <caption>` is merely that you could conceivably want indenting for this:

```js
/**
* @example <caption>
* Here is a long
* summary of this
* example
* </caption>
* ```js
* function hi () {
*   alert('Hello');
* }
* ```
*/
```

You might want `check-indentation` to apply to the caption contents (though you'd want the js block to instead be linted by normal `indent` rule as per #211).

But I don't personally care enough about this use case to think it warrants a fix/issue unless someone else did.

Reply:
@brettz9 thanks for the explanation. The way you explained it, it should work with the changes i made :). Just not by default, because example tag is excluded then. But if you disable excluding it, it will be checked and code will not because it's wrapped into fenced code block.
I can add that to the test assertions:
```js
    {
      code: `
          /**
          * @example <caption>
          * Here is a long
          * summary of this
          * example
          * </caption>
          * \`\`\`js
          * function hi () {
          *   alert('Hello');
          * }
          * \`\`\`
          */
      `,
      options: [{
        excludeTags: [],
      }],
    },
```
Now, if you would add some indentation inside the caption lines, it would error. Same if JS code would not be wrapped into fenced code block.

Reply:
Such an assertion sounds good, thanks!

Reply:
@brettz9 done: #392